packages/authhero/src/routes/universal-login/form-node.tsx

```
@@ -0,0 +1,145 @@
+      return ctx.html(
+        <FormComponent>
+          <h1>{form.name}</h1>
+          <div>{node.alias || node.type}</div>
+          {node.config &&
+            "components" in node.config &&
+            (node.config.components || []).map((comp: any) => {
+              if (comp.type === "RICH_TEXT") {
+                return (
+                  <div
+                    key={comp.id}
+                    dangerouslySetInnerHTML={{ __html: comp.config.content }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)